bokehjs/src/lib/models/widgets/tables/data_table.ts

```diff
@@ -134,11 +157,6 @@ export class DataTableView extends WidgetView {
     return [...super.styles(), slickgrid_css, tables_css]
   }
 
-  _update_layout(): void {
-    this.layout = new LayoutItem()
-    this.layout.set_sizing(this.box_sizing())
-  }
-
```

Comment:
Removing this seemed sufficient for the table to scroll correctly in responsive mode.

Reply:
This raises a question why this code was there and what its removal may break if anything. I this point I don't know/remember.

Reply:
Can only resolve this with lots of testing I guess.